packages/slice-machine/lib/builders/common/Zone/Card/components/Hints/Renderers/index.tsx

```diff
@@ -1,13 +1,15 @@
 import toVue from "./vue";
 import toReact from "./react";
 import toSvelte from "./svelte";
+import toPreviousReact from "./previous-react";
 
 export const svelte = toSvelte;
 
 export const vue = toVue;
-export const react = toReact;
+export const react = toPreviousReact;
```

Comment:
I think here you want to use to React as the new user will use the helpers lib :)

Reply:
Then that would be the new version

Reply:
yes that's what we want, no backward compatibility :)